_posts/blog/2018-11-07/2018-11-07-big-change-hard-work-and-strong-growth-openbazaar-2-turns-1.markdown

```diff
@@ -0,0 +1,53 @@
+
+#### Bitcoin Breakdown
+
+These changes took a long time to implement and deliver, but the team did so fairly seamlessly [on November 1st, 2017](https://openbazaar.org/blog/openbazaar-20-is-live/). There wasn’t much time to celebrate though as a transaction fee crisis was happening in parallel with a massive uptick in interest in Bitcoin. This sent the team into disarray as we struggled to find users who wanted to pay $30 in fees just to buy something like a $20 t-shirt in the marketplace. 
+
+After some consternation we decided the only solution was to expand our payment options to include other currencies to ease the scalability pains felt by our users and so in [January](https://openbazaar.org/blog/openbazaar-2-1-released-bitcoin-cash-and-zcash-integrated/) we released [Bitcoin Cash](https://www.bitcoincash.org/) and [ZCash (Beta)](https://z.cash) support for OpenBazaar. We are proud that we’ve been able to bring OpenBazaar to a broader community of crypto enthusiasts and are continuing that work today. 
+
+#### Wonder What's Next?
+
+In 2018 the breadth of crypto projects has increased exponentially through ERC-20 token issuances, completely new blockchains, layer two networks and more. We understand that there is a growing number of people interested in using all kinds of cryptocurrencies—not just Bitcoin—for e-commerce, coin trading, rewarding others for content creation and other use cases. OpenBazaar was always designed as a project for _the world_, not just for Bitcoin, and we’re hard at work re-engineering it to support new coins and communities. 
+
+Our next release will center on the [multiwallet](https://openbazaar.org/blog/Freedom-To-Trade-Means-Freedom-To-Choose-Currencies-The-OpenBazaar-Multiwallet/) which will allow users to send and receive payments from one wallet instance that can hold these five coins: Bitcoin, Bitcoin Cash, Zcash, Ethereum and Litecoin. We hope to introduce this new capability and others very soon.
+
+![OpenBazaar Multiwallet design](openbazaar-multiwallet-design.png "OpenBazaar Multiwallet design for Bitcoin, Bitcoin Cash, Zcash, Ethereum and Litecoin")
+> Want to see more? Click [here](https://projects.invisionapp.com/share/5XMZC8DT8HM#/screens/308962050_Openbazaar-2-0-Wallet) for an interactive demo!
+
+OB1 is also bringing OpenBazaar to mobile with **our own iOS and Android apps** and we’re pioneering the technology to make a truly decentralized web version of OpenBazaar so that anyone can run it in their own standard browser. It’s an extremely bright future and every day thousands of new people are learning about the power of Bitcoin, Ethereum and other crypto projects. 
```

Comment:
> and we’re pioneering 

I'd just start a new sentence here, with "We're pioneering" or "We're also pioneering".